support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/OAuth20Constants.java

```diff
@@ -345,4 +345,8 @@
      */
     String UNAUTHORIZED_CLIENT = "unauthorized_client";
 
+    /**
+     * The revocation url.
+     */
+    String REVOCATION_URL = "revoke";
```

Comment:
Move the constant next to all other URLs; Keep a consistent order as much as possible, and help the next reader.

Reply:
Done.